idunn/places/base.py

```diff
@@ -282,10 +283,18 @@ def get_meta(self):
 
     def get_bubble_star_url(self):
         if self.properties.get("ta:average_rating") is not None:
-            return (
+            rating = float(self.properties.get("ta:average_rating"))
+
+            base_url = (
                 f"https://www.tripadvisor.com/img/cdsi/img2/ratings/traveler/"
-                f"{self.properties.get('ta:average_rating')}-MCID-66562.svg"
+                f"{rating:.1f}-MCID-66562.svg"
```

Comment:
There was a small bug here as https://www.tripadvisor.com/img/cdsi/img2/ratings/traveler/4.5-MCID-66562.svg is a valid URL while https://www.tripadvisor.com/img/cdsi/img2/ratings/traveler/4-MCID-66562.svg isn't. So we have to ensure that the rating is formatted with exactly one digit after the decimal point.

Reply:
We can add a comment above for this trick : `Tripadvisor bubble star url need a rating with a decimal (e.g 4.0 or 4.5)`

Reply:
Nice catch btw :+1: